src/lib/utils/trim.cpp

```diff
@@ -0,0 +1,26 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   trim.cpp                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/11/27 16:38:54 by mcourtoi          #+#    #+#             */
+/*   Updated: 2024/02/06 12:54:21 by mcourtoi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include <string>
+#include <cctype>
+#include <iostream>
+
+// Fonction pour élaguer les espaces blancs au début et à la fin d'une chaîne de caractères
```

Comment:
What is this language?? Is this... French?!?! 🤢
Moreover, you must put a Doxygen comment here, not a C++ comment.

Reply:
Deleted the entire file as it's not used anymore